util/Setup/CertBuilder.cs

```diff
@@ -97,5 +97,19 @@ public void BuildForInstall()
                 Helpers.ShowBanner(_context, "WARNING", message, ConsoleColor.Yellow);
             }
         }
+
+        public void BuildForUpdater()
+        {
+            if (_context.Config.EnableKeyConnector && !File.Exists("/bitwarden/key-connector/bwkc.pfx"))
+            {
+                Directory.CreateDirectory("/bitwarden/key-connector/");
+                var KeyConnectorCertPassword = Helpers.GetValueFromEnvFile("key-connector",
```

Comment:
Does this value need to be set into the `_context.Install` state?

Reply:
I was thinking of the key connector as an optional addon to an already existing install with the added benefit of creating a small bit of friction for setting this up (not showing it in the CLI install wizard script)

Reply:
Sounds good to me, thanks for the clarification!